src/plugins/mavlink_ftp/fs.cpp

```diff
@@ -0,0 +1,116 @@
+#include <dirent.h>
+#include <fcntl.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <stack>
+#include <stdio.h>
+#include <unistd.h>
+
+#include "fs.h"
+
+#ifndef PATH_MAX
```

Comment:
This requires `#include <linux/limits.h>`, right?

Reply:
This ifndef was needed only for MacOS... so linux/limits.h was already included from somewhere.